SPM_Node.X/SPM.c

```diff
@@ -1,10 +1,167 @@
 #include "SPM.h"
 
+float analog_channels[36] = {0};
```

Comment:
In general I've read to pretty much never use floats for microcontroller stuff, esp 32-bit. Lots of weird rounding errors and junk